src/System Application/App/Email/permissions/EmailEdit.PermissionSet.al

```diff
@@ -21,6 +21,7 @@ permissionset 8901 "Email - Edit"
                   tabledata "Email Message" = imd,
                   tabledata "Email Message Attachment" = imd,
                   tabledata "Email Outbox" = imd,
+                  tabledata "Email Inbox" = id,
```

Comment:
No modify needed?

Reply:
Yes, this is on purpose. As we're retrieving the details from an external source, there should be no reason this record needs to be modified once it has be retrieved.